### DISCUSSION TITLE
Welcome to aitools_client Discussions!

### DISCUSSION THREAD

**Discussion by SethRobinson** [SethRobinson/aitools_client#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Aptronymist** [comment]
I just stumbled on this, I wanted to let you know that I like it a lot, I've been looking for a front-end that wasn't web-based, as they seem a lot more bloated, so good job!
The only criticism (constructive) would be that the actual UI for generation could use some tweaking or allowing some customization, like orientation or scaling, but this is a great project.
